xwiki-rendering-wikimodel/src/main/java/org/xwiki/rendering/wikimodel/tex/TexSerializer.java

```diff
@@ -42,13 +42,15 @@
 import org.xwiki.rendering.wikimodel.ReferenceHandler;
 import org.xwiki.rendering.wikimodel.WikiParameters;
 import org.xwiki.rendering.wikimodel.images.ImageUtil;
-
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 /**
  * @version $Id$
  * @since 4.0M1
  */
 public class TexSerializer extends PrintTextListener
 {
+    private static final Logger LOGGER = LoggerFactory.getLogger(TexSerializer.class);
     private static class DocumentContext
```

Comment:
You are missing an empty line after Logger declaration. Would be a pity to not follow XWiki codestyle for a [codestyle] improvement :)

If you use the Eclipse in the dev VM you can use Ctrl+Shift+F to properly format code (then make sure to commit only what's related to your work, sometime the file is not full following XWiki formatting rules in other places).